src/components/Search/SelectedRevisionItem.tsx

```diff
@@ -93,6 +93,7 @@ function SelectedRevisionItem({
                 <Link
                   href={getTreeherderURL(item.revision, repository)}
                   target='_blank'
+                  aria-label={`link to revision jobs in treeherder for ${revisionHash}`}
```

Comment:
As said above for a similar situation, we probably can use a title here, as this can be useful to sighted users.